_includes/content/reqs-sys.md

```diff
@@ -27,6 +27,14 @@
   > [Preventing NetworkManager from controlling {{site.prodname}} interfaces]({{ site.baseurl }}/maintenance/troubleshoot/troubleshooting#configure-networkmanager)
   > before installing {{site.prodname}}.
   {: .alert .alert-info}
+  
+- If distribution comes with installed Firewalld or another iptables manager it should be disabled. 
```

Comment:
"If your Linux distribution" 

Sorry, one more minor tweak I missed the first time.

Reply:
resolved